src/sdk/clients/createNexusClient.ts

```diff
@@ -217,11 +219,25 @@ export async function createNexusClient(
     attesterThreshold
   })
 
+  const paymasterContextFull = {
```

Comment:
Why is there a hardcoded token address in here? Please remove it.

Reply:
Resolved